webserver/pkgpkr/webservice/templates/webservice/base.html

```diff
@@ -1,78 +1,123 @@
-                <h5> Welcome {{ request.session.github_info.login }}</h5>
-            {% endif %}
-            <img style="width: 32px" src="{{ request.session.github_info.avatar_url }}">
+			gtag('config', 'G-3ZJTB9DHQG');
+		</script>
+  </head>
+  <body>
+    <section class="hero is-medium is-primary is-bold">
+      <div class="hero-body" style="padding-top: 4rem; padding-bottom: 2rem;">
+        <div class="container has-text-centered">
+          <a href="/">
+            <h1 class="title is-1">
+              <i class="fas fa-box"></i>
+              Package Picker
+            </h1>
+          </a>
+          <h2 class="subtitle">
+            Choose Good Packages, Faster.
+          </h2>
+        </div>
+        <nav class="container" role="navigation" aria-label="main navigation">
+          <div class="navbar">
+            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
+              <span aria-hidden="true"></span>
+              <span aria-hidden="true"></span>
+              <span aria-hidden="true"></span>
+            </a>
+          </div>
 
-        </form>
-    </div>
-</nav>
+          <div id="navbarBasicExample" class="navbar-menu">
+            <div class="navbar-start">
+              <a class="navbar-item" href="{% url 'index' %}">
+                <i class="fas fa-home"></i>
+                &nbsp;Home
+              </a>
 
-{% block nav %}
-{% endblock %}
+              <a class="navbar-item" href="{% url 'about' %}">
+                <i class="fas fa-question"></i>
+                &nbsp;About
+              </a>
 
-{% block body %}
-{% endblock %}
-</body>
+              {% if request.session.github_token %}
+              <a class="navbar-item" href="{% url 'repositories' %}">
+                <i class="fas fa-list"></i>
+                &nbsp;My Repositories
+              </a>
+              {% endif %}
+            </div>
 
+            <div class="navbar-end">
+              {% if request.session.github_token %}
+                {%  if request.session.github_info.name %}
+                <a class="navbar-item" style="pointer-events: none;">
+                  Welcome, {{ request.session.github_info.name }}
+                </a>
+                {%  else %}
+                <a class="navbar-item" style="pointer-events: none;">
+                  Welcome {{ request.session.github_info.login }}
+                </a>
+                {% endif %}
+                <img src="{{ request.session.github_info.avatar_url }}" alt="Profile" style="height: 48px; width: 40px; padding: 8px 0 0 0">
+              {% endif %}
+              <div class="navbar-item">
+                {% if not request.session.github_token %}
+                <div class="buttons">
+                  <a class="button is-linkl" href="{% url 'login' %}">
+                    <i class="fas fa-sign-in-alt"></i>
+                    <strong>&nbsp;Log in</strong>
```

Comment:
I think all the nbsp s could be substitute with adding padding for fas class throuhgout

Reply:
Let me see if I can do that